plugins/modules/aci_static_binding_to_epg.py

```diff
@@ -49,8 +49,8 @@
     description:
     - Determines the primary encapsulation ID associating the C(epg)
       with the interface path when using micro-segmentation.
-    - Accepted values are any valid encap ID for specified encap, currently ranges between C(1) and C(4096).
-    type: int
+    - Accepted values are any valid encap ID for specified encap, currently ranges between C(1) and C(4096) and 'unknown'.
```

Comment:
'unknown' -> C(unknown)
Can we also had a sentence that says that C(unknown) is the default value and that using C(unknown) disable the Micro-Segmentation?